handler/src/main/java/io/netty/handler/ssl/OpenSsl.java

```diff
@@ -198,4 +216,92 @@ static long memoryAddress(ByteBuf buf) {
     }
 
     private OpenSsl() { }
+
+    private static String normalizeOs(String value) {
```

Comment:
Agreed. How about updating the README.md of os-maven-plugin so it defines what the mapping is?